### DISCUSSION TITLE
Play AnimMontage

### DISCUSSION THREAD

**Discussion by Sheardo** [Sixze/ALS-Refactored#513] [General]
Hello,

I am trying to play an anim montage on an arm. 

![Screenshot 2024-05-29 141944](https://github.com/Sixze/ALS-Refactored/assets/58880956/afad1724-4b52-42a3-b21c-e159ea7f79be)

I am doing this with a simple key input for testing

![Screenshot 2024-05-29 141953](https://github.com/Sixze/ALS-Refactored/assets/58880956/c57cf770-9ad2-4056-9268-5e8c01dc3dc7)

I have assigned overlay layers for my animation within the anim montage

![Screenshot 2024-05-29 142010](https://github.com/Sixze/ALS-Refactored/assets/58880956/86621bda-4945-4e33-b058-eaf9128f50cf)

I have also used UAlsAnimationModifier_CreateLayeringCurves to add all the curves to the animation and keyed Right Arm to 1 for the entire animation.

https://github.com/Sixze/ALS-Refactored/assets/58880956/af109555-51e9-4412-8a71-676e27d99481

As you can see in the video, the animation doesn't play but the overlay is working. 

The AnimNotify for sound is working so the anim is being triggered.

I'm stuck on where I have gone wrong on this.

**Comment by Magic-Summer** [comment]
Hi @Sheardo
from what I see, you always use the same animation sequence. From these discussions, I gathered you additionally need mesh & local space versions:
[how layering works](https://github.com/Sixze/ALS-Refactored/discussions/19#discussioncomment-1348386)
[how curves work](https://github.com/Sixze/ALS-Refactored/discussions/196#discussioncomment-4272178)
I only have my curves on my "regular" animation sequence.
Good luck.
